.github/workflows/serverless-binary-size.yml

```diff
@@ -0,0 +1,99 @@
+name: "Serverless Binary Size"
+
+on:
+  pull_request:
+
+env:
+  SIZE_ALLOWANCE: 100000  # 0.1 MB
```

Comment:
Nit: this feels quite small (the current serverless binary size is ~32MB, so this will trigger for a 0.3% size increase), I'm worried that this will alert quite often. Is this something you are okay with (since you'll likely get quite a few questions and requests for help from this)?

Reply:
That's a great call out actually. I think a good thing to do in this situation is actually put this size allowance _larger_ than what we'd ideally want. This way it will only trigger for very large changes. As you mentioned, this would cut down on all the questions we would otherwise get.